### PR TITLE
docs: Fix schema.log_namespace and telemetry.tags documentation

### PR DESCRIPTION
**Pull request by spencergilbert**

### CODE REVIEW

File: website/cue/reference/configuration.cue [hunk 1]
```diff
@@ -277,7 +289,7 @@ configuration: {
 						description: """
 							Controls which tags should be included with the `vector_component_sent_events_total` and
 							`vector_component_sent_event_bytes_total` metrics.
-						"""
+							"""
```
Comment (jszwedko) [inline website/cue/reference/configuration.cue:292]: **👏 praise:** ‏ I noticed this the other day too. Thanks for fixing!

File: website/cue/reference/configuration.cue [hunk 2]
```diff
@@ -251,16 +251,28 @@ configuration: {
...
 		}
-		log_namespacing: {
-			common:      false
+		schema: {
+			common: false
 			description: """
-				Globally enables / disables log namespacing. See [Log Namespacing](\(urls.log_namespacing_blog))
-				for more details. If you want to enable individual sources, there is a config
-				option in the source configuration.
+				Configures options for how Vector handles schema.
```
Comment (jszwedko) [inline website/cue/reference/configuration.cue:257]: **💬 suggestion:** ‏ 
```suggestion
				Configures options for how Vector handles event schemas.
```

**Review by jszwedko — COMMENTED** [review]
It looks like the Netlify build failed due to one of the links needing to be updated. Otherwise looks good. I'll verify the deployed site once the link is fixed.

```
9:11:00 AM: blog/log-namespacing/index.html
9:11:00 AM:   hash does not exist --- blog/log-namespacing/index.html --> /docs/reference/configuration/global-options/#log_namespacing
```

File: website/cue/reference/configuration.cue (hunk 2 above)
Comment (spencergilbert) [reply website/cue/reference/configuration.cue]: [dc63455](https://github.com/vectordotdev/vector/pull/17961/commits/dc63455feaf8881145cb0ce78cfa9a09b76cbb35)

**Review by jszwedko — APPROVED** [review]

**Pull request merged**